src/components/views/create/CreateProject.tsx

```diff
@@ -366,7 +366,7 @@ const CreateProject: FC<ICreateProjectProps> = ({ project }) => {
 							<li>
 								{isEditMode ? 'Edited' : 'Newly published'}{' '}
 								projects will be &quot;unlisted&quot; until
-								reviewed by our team {isEditMode && 'again'}.
+								reviewed by our team{isEditMode && 'again'}.
```

Comment:
I think this is wrong
![image](https://user-images.githubusercontent.com/73441857/181191595-da0db894-f093-4e37-9cd3-6079f5ca4213.png)

Reply:
nice catch!